internal/compiler-bridge/src/main/scala/xsbt/API.scala

```diff
@@ -79,7 +79,6 @@ final class API(val global: CallbackGlobal) extends Compat with GlobalHelpers {
     def isTopLevel(sym: Symbol): Boolean = {
       !ignoredSymbol(sym) &&
       sym.isStatic &&
-      !sym.isImplClass &&
```

Comment:
I *think* all the appearances of `isImplClass` should not be removed because implementation classes were deprecated in 2.12, but it's still necessary to check for them in Scala 2.11 and 2.10.